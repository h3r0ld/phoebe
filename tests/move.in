start map.txt
move N
quit
