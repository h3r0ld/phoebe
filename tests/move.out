Player 0: Cell(22,8) Speed: 2 CanChangeDirection: True
Player 0: New Position: Cell(20, 8)